### ISSUE TITLE
Add option to keep/remove exiftool backup files after processing

### ISSUE THREAD

**Issue by berryp** [avdeveloper/import-fuji-settings#2]
By default, exiftool will backup each file before making changes to it. The default behaviour should be to delete these immediately (or not create them in the first place.) However, I can see where there may be occasions where users would like the option to keep them around. So having a flag in the plugin settings to store the backups - with an optional location? would be useful.

**Comment by albert-tse** [comment]
Yup a checkbox on the plugin settings would be the correct place to put it.
